client/mysql.cc

```diff
@@ -265,6 +265,7 @@ static char *current_host,*current_db,*current_user=0,*opt_password=0,
             *current_prompt=0, *delimiter_str= 0,
             *default_charset= (char*) MYSQL_AUTODETECT_CHARSET_NAME,
             *opt_init_command= 0;
+static char *opt_bind_address=0;
```

Comment:
technically `static` vars are always initialized to 0, so omit or use the consistent style of `opt_bind_address= 0;`

Reply:
fixed